assignment1/training.js

```diff
@@ -0,0 +1,67 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+function secondLargest(array) {
+  // Write your code here
+  max=Number.MIN_SAFE_INTEGER;
+  max2=Number.MIN_SAFE_INTEGER;
+  
+  for(i=0;i<array.length;i++)
+    {
+      if(array[i]>max) max=array[i];
+      
+   }
+  for(i=0;i<array.length;i++)
```

Comment:
@psinha20 Use following practises for code indentation  

```
for(i = 0; i < array.length ;i++)  {
}
```
- Use space after & before of every operator (=, <, > etc). i.e var a = 678
- Use space at the start & end of for loop. i.e for () {}
- Use  specific name as variable name. (don't use i)